packages/renderer/src/lib/kube/KubeActions.svelte

```diff
@@ -0,0 +1,7 @@
+<script lang="ts">
+import KubeApplyYamlButton from '../kube/KubeApplyYAMLButton.svelte';
+</script>
+
+<div class="flex flex-row gap-x-2">
```

Comment:
The additional-action section already defines button spacing, we shouldn't create a subcomponent that applies its own.

Reply:
You are right, I was not aware, fixed :+1: